src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -73,6 +74,19 @@ def sql_alter_from(self, catalog):
             f" TBLPROPERTIES ('upgraded_from' = '{self.key}');"
         )
 
+    def sql_unset_to(self, catalog):
+        return (
+            f"ALTER {self.kind} `{catalog}`.`{self.database}`.`{self.name}` "
+            f"UNSET TBLPROPERTIES IF EXISTS('upgraded_to');"
+        )
+
+    # SQL to reset the assessment record to revert migration state
+    def sql_unset_to_assessment(self, schema):
```

Comment:
Incorrect code: we can't rely on immutable assessment table as a mutable state for migration

Reply:
OK